brooklyn-tosca-transformer/src/main/java/io/cloudsoft/tosca/a4c/brooklyn/spec/RelationshipModifier.java

```diff
@@ -37,7 +41,10 @@ public void apply(EntitySpec<?> entitySpec, NodeTemplate nodeTemplate, Topology
             RelationshipTemplate relationshipTemplate = findRelationshipRequirement(nodeTemplate, requirementId);
             if (relationshipTemplate != null && relationshipTemplate.getType().equals("brooklyn.relationships.Configure")) {
 
-                Map<String, Object> relationProperties = getTemplatePropertyObjects(relationshipTemplate);
```

Comment:
I don't see the difficulty - the two implementations you have are:
```java
Map<String, PaaSNodeTemplate> builtPaaSNodeTemplates = treeBuilder.buildPaaSTopology(topology).getAllNodes();
String computeName = (nodeTemplate.getName() != null) ? nodeTemplate.getName() : (String) entitySpec.getFlags().get("tosca.template.id");
PaaSNodeTemplate paasNodeTemplate = builtPaaSNodeTemplates.get(computeName);
ConfigBag bag = ConfigBag.newInstance(getTemplatePropertyObjects(nodeTemplate, paasNodeTemplate, builtPaaSNodeTemplates));
```
and
```java

Map<String, PaaSNodeTemplate> builtPaaSNodeTemplates = treeBuilder.buildPaaSTopology(topology).getAllNodes();
String computeName = (nodeTemplate.getName() != null) ? nodeTemplate.getName() : (String) entitySpec.getFlags().get("tosca.template.id");
PaaSNodeTemplate paasNodeTemplate = builtPaaSNodeTemplates.get(computeName);
Map<String, Object> relationProperties = getTemplatePropertyObjects(relationshipTemplate, paasNodeTemplate, builtPaaSNodeTemplates);
```
They're identical but for `nodeTemplate` and `relationshipTemplate`, which are both `AbstractTemplates`.